oak_abi/src/label/mod.rs

```diff
@@ -66,8 +66,18 @@ impl crate::proto::oak::label::Label {
         // The target label must have (compared to the self label):
         // - same or more confidentiality tags
         // - same or fewer integrity tags
-        self_confidentiality_tags.is_subset(&other_confidentiality_tags)
-            && other_integrity_tags.is_subset(&self_integrity_tags)
+        (other_confidentiality_tags.contains(&Tag::universal_top_tag())
+            || self_confidentiality_tags.is_subset(&other_confidentiality_tags))
+            && (self_integrity_tags.contains(&Tag::universal_top_tag())
+                || other_integrity_tags.is_subset(&self_integrity_tags))
+    }
+}
+
+impl Tag {
+    /// Convenience function for creating top privilege tag that can declassify all types of tags.
+    pub fn universal_top_tag() -> Tag {
```

Comment:
Why do we need a method for this particular tag but not others? Or do we already have methods for the others elsewhere? I seem to remember we have constructors for various types of labels (though perhaps not tags).

Reply:
We have methods for creating other types of tags, but they are not members of `Tag`. They are ordinary functions. `top` is a special value, and felt more like `Label::public_untrusted`, so I had it as a method. Now changed it to be a function too.